dashboard/app/models/concerns/pd/jot_form_backed_form.rb

```diff
@@ -166,7 +166,6 @@ def get_form_id(category, name)
     # Useful for filling in placeholder response entries (submission id but no answers)
     def sync_from_jotform
       raise 'Missing submission id' unless submission_id.present?
-      self.class.sync_questions_from_jotform form_id
```

Comment:
This should have been removed as part of https://github.com/code-dot-org/code-dot-org/pull/22954